wdl-lint/DEFINITIONS.md

```diff
@@ -0,0 +1,46 @@
+# Definitions
+
+This document defines key concepts and terminology used throughout the WDL linting rules.
```

Comment:
as we are including this in other places that aren't this document, I would just axe this introduction and jump straight into the definitions
```suggestion
```